.github/workflows/ci.yml

```diff
@@ -215,8 +215,8 @@ jobs:
 
       - run: vendor/bin/phpcs -p -s
 
-  linting:
-    name: "Linting"
+  npm:
+    name: "NPM: Lint and unit test"
```

Comment:
Putting different linting tasks together is one thing, also including tests is another. I would not combine these. Nice to see the task level what type of failure you are dealing with.